lib/alephant/logger/json.rb

```diff
@@ -23,7 +25,7 @@ def initialize(log_path, options = {})
 
           hash = flatten_values_to_s(h) unless @nesting
 
-          write(hash)
+          write(hash) if write_level?(level)
```

Comment:
I would name this something like `should_write?` or `is_writeable?` to give a clearer idea of what the boolean return value represents?

Reply:
Good shout, IMO `writeable?` sounds more fitting.